tests/unit/test_general_options.py

```diff
@@ -1307,6 +1313,48 @@ class       Command group for CIM classes.
       'rc': 0,
       'test': 'innows'},
      None, OK],
+
+    #
+    # Test of 3 ways to execute same simple command. Creates a mock
+    # and tests combination of single command,
+    #
```

Comment:
These testcases depend on each other, i.e. on the previously generated connections file. That is a general problem, and we should not do that. See also issue #923.

Reply:
Why not. **DISCUSSION**

Reply:
Ok. Let's put this one in, since we have issue #923 for the .bak files.